src/main/java/org/folio/service/PasswordResetLinkServiceImpl.java

```diff
@@ -184,7 +186,7 @@ public PasswordResetLinkServiceImpl(ConfigurationClient configurationClient, Aut
   }
 
   @Override
-  public Future<TokenResponse> validateLinkAndLoginUser(OkapiConnectionParams okapiConnectionParams) {
+  public Future<Void> validateLinkAndLoginUser(OkapiConnectionParams okapiConnectionParams) {
```

Comment:
Seems like now it can be renamed to "validateLink"

Reply:
fix